### ISSUE TITLE
Docs: incorrect error span in documentation

### ISSUE THREAD

**Issue by DanielRosenwasser** [microsoft/TypeScript-Website#1527]
https://www.typescriptlang.org/docs/handbook/enums.html#union-enums-and-enum-member-types

```ts
enum ShapeKind {
  Circle,
  Square,
}

interface Circle {
  kind: ShapeKind.Circle;
  radius: number;
}

interface Square {
  kind: ShapeKind.Square;
  sideLength: number;
}

let c: Circle = {
  kind: ShapeKind.Square,
  radius: 100,
};
```

![image](https://user-images.githubusercontent.com/972891/105766726-f20c3800-5f0e-11eb-9a02-6bf15bb07c2c.png)

**Comment by orta** [comment]
I recently switched the shiki renderer to allow for a separation of different lines, this is doable now by moving the error underlines to live on their own div in-between the lines of source code.
